EIPS/eip-3074.md

```diff
@@ -1,6 +1,6 @@
 ---
 eip: 3074
-title: Native Sponsored Transactions
+title: AUTHORIZE and AUTHORIZEDCALL opcodes
```

Comment:
```suggestion
title: AUTHORIZEDCALL opcode
```

I prefer this, because it is the "meat" of the EIP. `AUTHORIZE` is really just a helper op.

Reply:
I think I prefer "AUTH and AUTHCALL opcodes", because those might grow to a family of opcodes in the future